orchestrator-salt/src/main/resources/salt/salt/unbound/init.sls

```diff
@@ -27,6 +27,10 @@ include_access_config:
   file.replace:
     - name: /etc/unbound/unbound.conf
     - pattern: '#include: "otherfile.conf"'
+{# Replace this, and any file.replace with sed}
+{# name: "sed -i 's/#include: \"otherfile\\.conf\"/include: \\\"\\/etc\\/unbound\\/access.conf\\\"/g' /etc/unbound/unbound.conf"}
+{# file.replace OR the unless check triggers a module load, which takes 5 seconds with an unpatched image, 2 seconds with a patch
```

Comment:
I won't skip unless, as if we don't have unless the state would always run, and salt would indicate it as if a change has happened

Reply:
Not planning to skip the unless check. Just seems like a combination of file.replace pattern along with unless causes all modules to be loaded.